app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+  
+  	def show
```

Comment:
```suggestion
  def show
```
sửa lại format code cho dễ nhìn nhé